decoder.rb

```diff
@@ -0,0 +1,47 @@
+MORSE_CODE = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}.freeze
+
+def split_word(string)
+  string.split
+end
+
+def decode(string)
+  word = ''
+  split_word(string).each do |c|
+    word += MORSE_CODE[c]
+  end
+  "#{word} "
+end
+
+INPUT = '.-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...'.freeze
+output = ''
+INPUT.split('   ').each do |s|
+  output += decode(s)
+end
+puts output
```

Comment:
Please, let adhere to the project core requirements as much as possible, According to the [requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/morse_code.md) please implement the following 😊

- Create a method to decode a Morse code character
- Create a method to decode an entire word in Morse code
- Create a method to decode the entire message in Morse code
- Reuse methods whenever possible

Running the lines of code below from the terminal

`p decode(".-   -... --- -..-   ..-. ..- .-.. .-..    --- ..-.    .-. ..- -... .. . ...")`

❗ Result got 
![image](https://github.com/aamir-asaram/morse-decoder/assets/78251927/145d6e6b-c3b5-4d66-96e1-7ce77a3077ca)

Expected (space between words)
`"A BOX FULL OF RUBIES"`

Please correct this. 🛠👍